shared-functions.sh

```diff
@@ -99,7 +97,7 @@ install_protoc_common() {
     # function. This file is platform dependent.
     zip_url=$1
 
-    # We use protocol buffers in webapp's event log stream infrastructure. This
+    # We use protocol buffers in webapp`'s event log stream infrastructure. This
```

Comment:
Typo?

Reply:
Yep, I have a hotkey of cmd+\` to open my terminal so I tend to accidentally sprinkle random \`s around. Oops!